python/basix/ufl_wrapper.py

```diff
@@ -1168,8 +1170,20 @@ def create_tensor_element(
     return TensorElement(e)
 
 
-def _create_enriched_element(elements: _typing.List[_FiniteElementBase]) -> _FiniteElementBase:
+def create_enriched_element(
+    elements: _typing.List[_FiniteElementBase], preserve_functions: bool = None, preserve_dofs: bool = None
+) -> _FiniteElementBase:
     """Create an enriched element from a list of elements."""
+    if preserve_dofs is None and preserve_functions is None:
```

Comment:
It would be beneficial for others if the doc string explained what the new inputs did (as this is no longer an internal function)